perllib/FixMyStreet/Cobrand/Bristol.pm

```diff
@@ -188,15 +188,23 @@ sub munge_overlapping_asset_bodies {
     my ($self, $bodies) = @_;
 
     my $all_areas = $self->{c}->stash->{all_areas};
-    my $in_area = scalar(%$all_areas) == 1 && (values %$all_areas)[0]->{id} eq $self->council_area_id->[0];
-    return if $in_area;
 
-    # If we get here, assume we are outside Bristol boundaries.
-    if ($self->check_report_is_on_cobrand_asset) {
-        # The report is in a park that Bristol is responsible for, so only show Bristol categories.
+    if (grep ($self->council_area_id->[0] == $_, keys %$all_areas)) {
+        # We are in an area covered by Bristol so only show Bristol categories and any multi-area cateories.
```

Comment:
We're not in an area covered by Bristol here, we are within the area of Bristol here. So I think if that's the case you can simply `return`, and not change anything at this point - unlike Brent/Camden, it's one way, Bristol covers two surrounding areas, but nothing covers Bristol.